packages/react-native-bpk-component-text-input/package.json

```diff
@@ -16,8 +16,9 @@
   "dependencies": {
     "bpk-tokens": "^27.4.7",
     "prop-types": "^15.5.8",
-    "react-native-bpk-component-icon": "^1.14.5",
-    "react-native-bpk-component-text": "^4.1.3",
+    "react-native-bpk-component-icon": "^1.14.4",
+    "react-native-bpk-component-text": "^4.1.2",
```

Comment:
It looks like these versions have regressed - icon should be `^1.1.14` and text should be `^4.1.3`.